wire-library/wire-tests/src/commonTest/proto/kotlin/map.proto

```diff
@@ -21,6 +21,19 @@ message Mappy {
   map<string, Thing> things = 1;
 }
 
+message MappyTwo {
```

Comment:
This may technically be proto syntax 3 ?
That said, iOS refuses to emit proto3 maps for whatever reason (I don't see anything in SwiftGenerator)